docs/semgrep-assistant/overview.md

```diff
@@ -86,6 +86,14 @@ This information is also available in Semgrep Cloud Platform on the **Dashboard*
 ![Semgrep Assistant's priority inbox email and Dashboard view](/img/priority-inbox.png#md-width)
     *Figure*. Semgrep Assistant's priority inbox Dashboard view.
 
+### Custom rules editor
+
+Semgrep Assistant can help you write custom rules to find patterns and vulnerabilities specific to your codebase. The only information you need to provide are the following:
```

Comment:
The good code and bad code are not necessary to produce an output. They are optional fields that will be used for additional tests and to provide context to the llm